linux_os/guide/auditing/auditd_configure_rules/audit_dac_actions/audit_rules_dac_modification_chmod/tests/ocp4/e2e.yml

```diff
@@ -1,3 +1,4 @@
 ---
-default_result: FAIL
-result_after_remediation: PASS
+# approach until we build better testing support for multi-arch assertions
```

Comment:
Couldn't we update all these in the `tests/assertions/ocp` directory? These `e2e.yml` files only get used as a last resort, don't they?

Reply:
oops, let me update those in `tests/assertions/ocp`